cmd/influx/query.go

```diff
@@ -23,31 +22,20 @@ or execute a literal Flux query contained in a file by specifying the file prefi
 }
 
 var queryFlags struct {
-	OrgID string
-	Org   string
+	organization
```

Comment:
I'm not wild about this semantically, and I'm not sure there's much benefit to encapsulating OrgID/Org if they're going to be accessed directly. Just makes it a little harder to read/understand.

Reply:
fair point, I tried to tread really lightly when I added this but crossed over boundaries a little. on the flip, this does DRY up a lot of cobra stuff across the board, and organization lookups by name all go through one code path [here](https://github.com/influxdata/influxdb/blob/6fb1b5bfa09d1bf6b2a1b42cdce7e70f251c0a11/cmd/influx/main.go#L254). I'm open to suggestions on where to put this as well

Reply:
I agree it's a little hard to understand at beginning. But overall, it would help us to make all of our flags more consistent. So still worth it in my opinion. Ideally, CLI should inherent all the behavior of the API, plus some interactive feature and file parsing.